Makefile

```diff
@@ -0,0 +1,52 @@
+MAJOR = $(shell grep "^\#define[[:space:]]*MAJOR" addons/main/script_mod.hpp | egrep -m 1 -o '[[:digit:]]+')
+MINOR = $(shell grep "^\#define[[:space:]]*MINOR" addons/main/script_mod.hpp | egrep -m 1 -o '[[:digit:]]+')
+PATCH = $(shell grep "^\#define[[:space:]]*PATCHLVL" addons/main/script_mod.hpp | egrep -m 1 -o '[[:digit:]]+')
+BUILD = $(shell grep "^\#define[[:space:]]*BUILD" addons/main/script_mod.hpp | egrep -m 1 -o '[[:digit:]]+')
+VERSION = v$(MAJOR).$(MINOR).$(PATCH).$(BUILD)
+PREFIX = cba
+BIN = @CBA_A3
+ZIP = CBA_A3
+ARMA = include
```

Comment:
Explain what this is - there is no folder named `include` in the CBA repository.